r/R/atac_utils.R

```diff
@@ -431,21 +431,21 @@ write_insertion_bed <- function(fragments, path,
   } else {
     assert_len(cell_groups, length(cellNames(fragments)))
     cell_groups <- as.factor(cell_groups)
+    levels(cell_groups) <- normalize_unique_file_names(levels(cell_groups))
```

Comment:
I was moreso thinking on the side of having this still function independently.  ie for some reason `write_insertion_bed()` was being used separately from `call_macs_peaks()` it will still be able to work with haphazardly formed cell_group names.

Reply:
I think the code as is actually breaks users with weird `cell_group` names rather than fixing them -- `path` is already supposed to hold valid path names, and the responsibility for that is very clearly on the user. But if we mess with `levels(cell_groups)`, then it might not line up anymore with the names of `path`.